monkey/tests/unit_tests/infection_monkey/utils/test_commands.py

```diff
@@ -0,0 +1,111 @@
+from infection_monkey.utils.commands import (
+    build_monkey_commandline_explicitly,
+    get_monkey_cmd_lines_linux,
+    get_monkey_cmd_lines_windows,
+)
+
+
+def test_build_monkey_commandline_explicitly():
```

Comment:
This test looks like 3 different tests all rolled into one function. It should be 3 different functions with names that describe what's being tested.